include/jemalloc/internal/arena_stats.h

```diff
@@ -50,6 +50,19 @@ struct arena_stats_decay_s {
 	arena_stats_u64_t	purged;
 };
 
+typedef struct arena_stats_extents_s arena_stats_extents_t;
+struct arena_stats_extents_s {
+	/* Total number of extents/bytes in extents_dirty. */
+	atomic_zu_t ndirty;
```

Comment:
I don't remember off the top of my head if these can end up getting touched from multiple threads; @interwq, do you?